registry/handlers/mail.go

```diff
@@ -0,0 +1,45 @@
+package handlers
+
+import (
+	"errors"
+	"net/smtp"
+	"strings"
+)
+
+// Mail provides fields of email configuration for sending.
+type Mail struct {
```

Comment:
@stevvooe 
It is used as one field's struct of `LogHook`. If we don't export it and name it 'mailer', the field can't be set out of package, and `LogHook` will be unable to used out of `handler` package. Do you really want that?
Even following what you said, I have to change other field  and LogHook's name to little letter. Right?